docs/config/tag-defs/this.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  name: 'this'
```

Comment:
I tried adding that and I get a ton of warnings like:

```
warn:    Missing container document: "$anchorScrollProvider" - doc "$anchorScrollProvider#method:disableAutoScrolling" (method)  - from file "ng/anchorScroll.js" - starting at line 16, ending at line 28
warn:    Missing container document: "$cacheFactory" - doc "$cacheFactory#method:info" (method)  - from file "ng/cacheFactory.js" - starting at line 320, ending at line 328
...
warn:    Missing service "module:ng.service:$anchorScrollProvider
@this" for provider - doc "module:ng.provider:$anchorScrollProvider
@this" (provider)  - from file "ng/anchorScroll.js" - starting at line 3, ending at line 11
warn:    Missing service "module:ng.service:$controllerProvider
@this" for provider - doc "module:ng.provider:$controllerProvider
@this" (provider)  - from file "ng/controller.js" - starting at line 16, ending at line 27
...
warn:    Invalid link (does not match any doc): "ngRoute.$routeProvider" - doc "index" (overview)  - from file "api/index.ngdoc" - starting at line 1
warn:    Invalid link (does not match any doc): "ngAria.$ariaProvider" - doc "index" (overview)  - from file "api/index.ngdoc" - starting at line 1
...
```

And also couple of errors at the end:

```
error:   Error: ENOENT: no such file or directory, open 'C:\Datafiles\Dev\AngularJS\angular.js\build\docs\partials\api\ng\provider\$anchorScrollProvider
@this.html'
    at Error (native)
error:   Error processing docs:  Error: ENOENT: no such file or directory, open 'C:\Datafiles\Dev\AngularJS\angular.js\build\docs\partials\api\ng\provider\$anchorScrollProvider
@this.html'
    at Error (native)
```

Thoughts?